api/controllers/AuthController.js

```diff
@@ -166,7 +166,7 @@ module.exports = {
         }
 
         // Mark the session as authenticated to work with default Sails sessionAuth.js policy
-        req.session.authenticated = true
+        req.session.authenticated = true;
```

Comment:
We don't have the `semi` rule enabled. I was going to enable it, but we're also missing a few semicolons in the clientside code and I didn't want to touch that for now.